core/rfb.js

```diff
@@ -148,16 +161,23 @@
         'target': 'null',                       // VNC display rendering Canvas object
         'focusContainer': document,             // DOM element that captures keyboard input
         'encrypt': false,                       // Use TLS/SSL/wss encryption
-        'true_color': true,                     // Request true color pixel data
+        'convert_color': false,                  // Client will not honor request for native color
         'local_cursor': false,                  // Request locally rendered cursor
         'shared': true,                         // Request shared mode
         'view_only': false,                     // Disable client mouse/keyboard
+        'aten_password_sep': ':',               // Separator for ATEN iKVM password fields
         'xvp_password_sep': '@',                // Separator for XVP password fields
```

Comment:
I don't like piling on more stuff we don't really want. At minimum you should use the same separator.